src/scss/modules/frame.module.scss

```diff
@@ -25,6 +25,14 @@
     margin: 0;
   }
 
+  &__noPadding {
```

Comment:
Jeg skjønte kanskje ikke helt denne her.
`__noPadding` selector har samme padding som `__hasPadding`, med litt ekstra i bunnen i tillegg. (kanskje er navnet litt forvirrende?)
Så ser jeg at den er kun i bruk i `Card`, så kanskje burde den kodes direkte i Card isåfall. (så kan vi evt. implementere det her om vi ser at det er behov for gjenbruk)
Når jeg ser videre på Landingssiden, forstod jeg ikke hvorfor Card ble brukt når VStack brukes. Hvis det er for responsiviteten, ville jeg kanskje tro at det er enklest å kode en enkel <section> med følgende styles for å få responsiviteten:
 composes: innerframe from '../../../scss/modules/frame.module.scss';
  composes: whitesection from '../../../scss/modules/fadein.module.scss';
  Så koder man padding og evt andre justeringen på den (uten bruk av Card eller utvidelse av Frame og Card) 
  Mulig jeg misforstod hensikten med det også, så vi kan ta en prat hvis ikke det ga mening